src/main/java/org/example/model/Player.java

```diff
@@ -0,0 +1,37 @@
+package org.example.model;
+
+import java.util.ArrayList;
+
+public class Player extends House {
+
+    private final String nick;
+    private int playerPoints = 0;
+    private boolean playerWin;
+
+    public Player(String nick) {
+        this.nick = nick;
+        playerWin = false;
+        new ArrayList<>();
```

Comment:
to be deleted?

Reply:
I forgot to delete it. Initially, Player and House were independent classes.

Reply:
fixed